starter-code/index.js

```diff
@@ -1,11 +1,69 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor(items, length) {
```

Comment:
you didn't need to pass anything into the constructor, because you never use them